### PR TITLE
feat: adding mentions about forking back after transferring repos

### LINKED ISSUE #13044: "Transferring a repository" could mention something about forking back after transfering
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

### What part(s) of the article would you like to see updated?

I've already transferred a repository. The documentation states "All links to the previous repository location are automatically redirected to the new location.", which is working great. However, now I would like to know if I can create a fork back to the original account, without destroying those redirects.

Intuitively I would think, that the redirects will be lost entirely, when I would simply fork the repository, but it would be helpful if the docs could confirm this.

As a workaround to this, I am thinking about forking to a temporary account first, then renaming the repository (since it doesn't seem possible to rename while forking), and then transferring the renamed repository to the original account. Before I do this I would like to be sure that the redirects will keep working for the original name, and I think similarly it would be helpful if the docs could include some information about this.

### Additional information

---
### Content plan
Check out the [content plan](https://github.com/github/docs/issues/13044#issuecomment-1141891922) for guidance.

### PR DESCRIPTION
**Pull request by CBID2**
### Why:
Per, [@stevecat's recommendation](https://github.com/github/docs/issues/13044#issuecomment-1141891922), I added information more information about redirects. This helps owners of repositories gain more awareness about what could occur when they transfer ownership over their repos to other people.  
Closes #13044 

### What's being changed (if available, include any code snippets, screenshots, or gifs):
screenshot of changes

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @cmwilson21! :) I made the PR.

**Review by bjorn — COMMENTED** [review]
Just a suggestion for improvement.

Note that I haven't verified whether what @stevecat wrote was the actual behavior. One thing strikes me as a little odd, which is that when I have previously renamed a repository, causing redirects to be created, and then overrode those redirects with a new repository, and then rename the repository again, I might actually want the redirects to be _changed to point to the location of the second rename_. But if the behavior is as described, the first redirects will always get re-activated and there is no (documented) way of changing the redirects.

File: content/repositories/creating-and-managing-repositories/transferring-a-repository.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ When you transfer a repository, its issues, pull requests, wiki, stars, and watc
   {% endwarning %}
 
 - When you transfer a repository from an organization to a personal account, the repository's read-only collaborators will not be transferred. This is because collaborators can't have read-only access to repositories owned by a personal account. For more information about repository permission levels, see "[AUTOTITLE](/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository)" and "[AUTOTITLE](/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization)."{% ifversion fpt or ghec %}
+- Redirects will function until someone else creates a new repository or fork with the same name.
+- Renaming or deleting the transferred repository or fork will reinstate the redirect.
```
[the remark this answers is not in the document]
Comment (CBID2) [reply content/repositories/creating-and-managing-repositories/transferring-a-repository.md]: Hi @bjorn! :) Thanks for clarifying! :) I made the necessary change.

File: content/repositories/creating-and-managing-repositories/transferring-a-repository.md [hunk 2]
```diff
@@ -58,10 +58,9 @@ When you transfer a repository, its issues, pull requests, wiki, stars, and watc
 
   {% warning %}
 
-  **Warning**: If you create a new repository under your account in the future, do not reuse the original name of the transferred repository. If you do, redirects to the transferred repository will no longer work.
+  **Warning**: If you create a new repository at the previous repository location, the redirects to the transferred repository will be deactivated. They can be reactivated by renaming or deleting the new repository.
```
Comment (stevecat) [inline content/repositories/creating-and-managing-repositories/transferring-a-repository.md:61]: I'm just adding "or fork" here to be extra clear.

```suggestion
  **Warning**: If you create a new repository or fork at the previous repository location, the redirects to the transferred repository will be deactivated. They can be reactivated by renaming or deleting the new repository or fork.
```

**Review by guntrip — DISMISSED** [review]
Thank you for working on this, @CBID2! ✨ And a great suggestion from @bjorn. 

I made one tiny suggested change, adding "or fork", and I'll apply that myself before merging.

The failing GraphQL test isn't anything to do with your changes, I suspect it'll pass when `main` is merged in as we deploy, but otherwise I'll troubleshoot this with the engineering team.

**Review by guntrip — APPROVED** [review]

**Comment by stevecat** [comment]
Boooo! 😞 This is an issue on our end, I've raised it with our engineering team and we'll get this merged as soon as we can!

**Comment by CBID2** [comment]
> Boooo! 😞 This is an issue on our end, I've raised it with our engineering team and we'll get this merged as soon as we can!

Did I do something wrong @stevecat? I'm confused.

**Pull request merged**